testsuite/features/step_definitions/common_steps.rb

```diff
@@ -811,23 +813,25 @@
   os_version, os_family = get_os_version($build_host)
 
   # Distribution
-  repos = "os_pool_repo os_update_repo"
+  repos = $is_cloud_provider ? OS_REPOS_BY_OS_VERSION[os_version].join(' ') : "os_pool_repo os_update_repo"
   log $build_host.run("zypper mr --disable #{repos}")
 
   # Tools
-  repos, _code = $build_host.run('zypper lr | grep "tools" | cut -d"|" -f2')
-  log $build_host.run("zypper mr --disable #{repos.gsub(/\s/, ' ')}")
+  unless $is_cloud_provider
```

Comment:
this is more of a personal preference but for readibility sake, is there a reason why we're using unless instead of the `if !$is_cloud_provider` used in other places? if it's indifferent, I think it's easier to understand if we use if instead of unless in these kinds of conditions

Reply:
I don't have opposition to change to if.

Reply:
Rubocop will :P 
Rubocop prefers unless in this case, to avoid negations.